src/commands/search_log_by_text.ts

```diff
@@ -16,20 +15,50 @@ export class SearchLogByText extends Command {
       return;
     }
 
-    try {
-      const resource = toSvnUri(
-        Uri.file(repository.workspaceRoot),
-        SvnUriAction.LOG_SEARCH,
-        { search: input }
-      );
-      const uri = resource.with({
-        path: path.join(resource.path, "svn.log")
+    const uri = Uri.parse("svnfs:/svn.log");
+    svnFs.writeFile(uri, Buffer.from(""), {
+      create: true,
+      overwrite: true
+    });
+
+    await commands.executeCommand<void>("vscode.open", uri);
+
+    const proc = cp.spawn("svn", ["log", "--search", input], {
+      cwd: repository.workspaceRoot
+    });
+
+    let content = "";
+
+    proc.stdout.on("data", data => {
+      content += data.toString();
+
+      svnFs.writeFile(uri, Buffer.from(content), {
+        create: true,
+        overwrite: true
       });
+    });
 
-      await commands.executeCommand<void>("vscode.open", uri);
-    } catch (error) {
-      console.error(error);
-      window.showErrorMessage("Unable to log");
-    }
+    window.withProgress(
+      {
+        cancellable: true,
+        location: ProgressLocation.Notification,
+        title: "Searching Log"
+      },
+      (_progress, token) => {
+        token.onCancellationRequested(() => {
+          proc.kill("SIGINT");
+        });
+
+        return new Promise((resolve, reject) => {
+          proc.on("exit", () => {
```

Comment:
I think it's worth checking for nonzero code (first callback argument, number)

Reply:
done